api/proto/config/appconfig.proto

```diff
@@ -110,6 +110,10 @@ message AppInstanceConfig {
 
   // metadata type to use for app if provided inside userData
   MetaDataType metaDataType = 17;
+
+  // profile_list is a set of strings which can be used to control which sets
+  // of applications get halted.
+  repeated string profile_list = 18;
```

Comment:
I think, it is reasonable to describe behavior when the list is empty.

Reply:
Good point. I guess if an app part of no profile, and a default_profile is set, it makes sense to have that application match?? Basically "empty" means wildcard - will run for any profile.